src/payment/models/history/history.controller.ts

```diff
@@ -41,6 +41,9 @@ export class HistoryController {
     @Query() query: HistoryQuery,
   ): Promise<HistoryDto[] | CoinTrackingHistoryDto[]> {
     return await this.historyService.getHistory(jwt.id, jwt.address, query);
```

Comment:
Is there something missing? We're not calling the BuyCrypto and BuyFiat Service? 🤔